Program_1.0.py

```diff
@@ -11,13 +15,56 @@
 client = MongoClient()
 db = client.countries
 
+host_name = 'localhost'
+post_number = 9901
+
+
+class WikiHandler(BaseHTTPRequestHandler):
+    def do_POST(self):
+        self.send_response(200)
+        self.end_headers()
```

Comment:
nie mam pewności i nie wiem czy to coś zmienia ale pewnie przy odsyłaniu przed end headers bedziesz potrzebowal  self.send_header("Content-type", "text/html;charset=utf-8")